riscv/tests/riscv_data/std_hello_world/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "std_hello_world"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+powdr-riscv-syscalls = { path = "../../../../riscv-syscalls" }
+powdr-riscv-runtime = { path = "../../../../riscv-runtime", features = ["std"]}
+
+[profile.release]
+panic = 'abort'
+
+[profile.dev]
+panic = 'abort'
```

Comment:
Are these needed or are you just testing it?

Reply:
This is either needed here or at the `RUSTFLAGS` during build. Maybe it is better to put it there, together with the rest of the crazy flags we use.

Reply:
Yea agree, otherwise the user has to do it themselves